src/routes/download/+page.svelte

```diff
@@ -1,37 +1,42 @@
 <script lang="ts">
 	import { links } from "$data/links";
 	import { defaultI18nValues, externalLink, Metadata } from "$lib";
-	import { Button, InfoBadge, InfoBar, TextBlock } from "fluent-svelte";
+	import { Button, InfoBar, TextBlock } from "fluent-svelte";
 	import { onMount } from "svelte";
 	import DownloadSourceCard from "./DownloadSourceCard.svelte";
 	import type { DownloadSource } from "./types";
 	import { _ } from "svelte-i18n";
 
 	let isWindows = false;
+	let innerWidth = 649;
 
 	const downloadSources = [
 		{
 			name: $_("download.microsoft_store.title", defaultI18nValues),
 			description: $_(
 				"download.microsoft_store.description",
-				defaultI18nValues
+				defaultI18nValues,
 			),
-			href: `ms-windows-store://pdp/?ProductId=9nghp3dx8hdx&cid=FilesWebsite`,
+			href: `https://apps.microsoft.com/detail/9nghp3dx8hdx?mode=full`,
```

Comment:
Aside from no longer opening the Microsoft Store, this change removes the campaign data. Can you please revert it for now?

Reply:
@Lamparter